ext/opentelemetry-ext-jaeger/tests/test_jaeger_exporter.py

```diff
@@ -34,7 +35,9 @@ def setUp(self):
             is_remote=False,
         )
 
-        self._test_span = trace.Span("test_span", context=context)
+        self._test_span = trace.Span(
```

Comment:
if you always attach an empty resource by default, you're omitting the valid test case of a Span with "None" as the resource.

For the purposes of this PR, I recommend adding a case that assumes the Span.resource may be None.

Added #702 for a more global fix.

Reply:
Thanks for reviewing @toumorokoshi, I made the requested changes.
Do you think I should add another test case explicitly handling the `None` case?
Or is it good enough that it's contained in the other tests?